packages/action-menu/test/index.ts

```diff
@@ -304,6 +304,15 @@ export const testActionMenu = (mode: 'sync' | 'async'): void => {
             expect(button).to.have.attribute('aria-expanded', 'true');
             expect(button).to.have.attribute('aria-controls', 'menu');
         });
+        it('has attribute aria-describedby', async () => {
```

Comment:
Great first step, but what we really need here is a test leveraged [findDescribedNode](https://github.com/adobe/spectrum-web-components/blob/main/test/testing-helpers-a11y.ts#L21) to confirm the expected association is being created here.

Reply:
Done!